bundles/org.openhab.binding.sony/src/main/resources/ESH-INF/thing/scalar.xml

```diff
@@ -0,0 +1,1480 @@
+		<description>General Setting for a string</description>
+	</channel-type>
+
+	<channel-type id="scalaraudiogeneralsettingdimmer" advanced="true">
+		<item-type>Dimmer</item-type>
+		<label>General Setting Dimmer</label>
+		<description>General Setting for a dimmer</description>
+	</channel-type>
+
+	<channel-type id="scalarsystemcurrenttime" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Current Time</label>
+		<description>Current time of the device</description>
+	</channel-type>
+
+	<channel-type id="scalarsystemlanguage" advanced="true">
+		<item-type>String</item-type>
+		<label>System Language</label>
+		<description>The current language of the device</description>
+	</channel-type>
+
+	<channel-type id="scalarsystemledindicatorstatus" advanced="true">
+		<item-type>String</item-type>
+		<label>LED Status</label>
+		<description>The status of the LED indicator (Off/Low/High)</description>
+	</channel-type>
+
+	<channel-type id="scalarsystempowersavingmode" advanced="true">
+		<item-type>String</item-type>
+		<label>Power Savings Mode</label>
+		<description>The power savings mode (Off/Low/High)</description>
+	</channel-type>
+
+	<channel-type id="scalarsystempowerstatus">
+		<item-type>Switch</item-type>
+		<label>Power Status</label>
+		<description>The power status of the device</description>
+	</channel-type>
+
+	<channel-type id="scalarsystemwolmode" advanced="true">
+		<item-type>Switch</item-type>
+		<label>WOL Mode</label>
+		<description>The wake on lan (WOL) mode</description>
+	</channel-type>
+
+	<channel-type id="scalarsystemircc">
+		<item-type>String</item-type>
+		<label>IRCC Command</label>
+		<description>The IRCC command to send to the system</description>
+	</channel-type>
+
+	<channel-type id="scalarsystemreboot" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Device Reboot</label>
+		<description>Send 'ON' to reboot the device</description>
+	</channel-type>
+
+	<channel-type id="scalarsystempostalcode" advanced="true">
+		<item-type>String</item-type>
+		<label>Postal Code</label>
+		<description>The postal code of the device</description>
+	</channel-type>
+
+	<channel-type id="scalarappcontrolapptitle" advanced="true">
+		<item-type>String</item-type>
+		<label>App Title</label>
+		<description>App title</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarappcontrolappicon" advanced="true">
+		<item-type>Image</item-type>
+		<label>App Icon</label>
+		<description>App Icon</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarappcontrolappdata" advanced="true">
+		<item-type>String</item-type>
+		<label>App Data</label>
+		<description>App Data</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarappcontrolappstatus" advanced="true">
+		<item-type>String</item-type>
+		<label>App Command</label>
+		<description>The application command</description>
+		<state>
```

Comment:
Is this a command channel, or does it also keep the state? Otherwise it should be a command channel:
https://www.openhab.org/docs/developer/bindings/thing-xml.html#command-description

Reply:
Yeah - command channels were added later than this binding and I never adapted to them because I use a DynamicStateProvider to provide state for both commands and actual state.  If I switch to commands - then I need to create a whole parallel system to handle dynamic command - wish whomever implemented the dynamic command would have brought them together somehow so they can be treated the same.  Let me look into how difficult it would be to do that..

Reply:
The other real issue is the serialization/deserialization of existing thing types - I have about 200 thing types registered right now and if I switch to commands, I'd need to modify all 200 and figure out some type of transition for existing users (blech)

Reply:
After reviewing - was much simplier than I envisioned.  The only two channels that were affected was the application status (as you noted) and the av content cmd channels - and those aren't serialized or anything - so updates made